docs/download/data-access.md

```diff
@@ -24,34 +24,13 @@ As a prerequisite to applying for DACO access, please ensure that you:
-
-1. Once the application is complete, you can submit it by “finalizing your DACO Application”. This action will prevent you from making any further changes to your application without authorization from an administrator.
-1. You will receive an email with further instructions. The next steps will also be listed on the confirmation page. Download and print your finalized application and add the necessary signatures. Once signed, you can fax or mail the application to the DACO office (the contact details are provided on the confirmation page and email).
-1. The DACO office will notify you when your application has been approved, or if they require any further information.
-1. Once your DACO access application is approved, your “OpenID” Google email address will become associated with an ICGC account and you will be granted access to the ICGC controlled access datasets available through [Cancer Genome Collaboratory](repositories/#collaboratory), [AWS Virginia](repositories/#aws), [Azure](repositories/#azure), the EBI’s [EGA repository](repositories/#ega) and [ICGC Data Portal](https://dcc.icgc.org/releases). You will receive an email from EGA about setting up your password to access controlled data at [https://ega-archive.org/](https://ega-archive.org/). If you already had an EGA account from before, you will use the same username/password to access ICGC controlled data at EGA.
+The instructions for applying for DACO can be found here on the [ICGC DACO website](https://daco.icgc-argo.org/).
 
 ![Applying-to-DACO](images/Applying_to_DACO_Diagram.png)
 
 <span style="color:red">***Please note:***</span> EGA is not responsible for granting access to ICGC data hosted at EGA.
 
 
-## The DACO Approval Process
-
-The DACO approval process takes approximately 24-48 hours.
-
-Your research project will be checked for conformity with the goals and [policies of ICGC ARGO](https://www.icgc-argo.org/page/72/introduction-and-goals-) including, but not limited to, policies concerning the purpose and relevance of the research, the protection of the participants and the security of the participants' data.
-
-The DACO office will notify you by email when your DACO application has been approved, or if they require any further information. Upon approval, access to controlled data will be granted for a **one year period** (starting from the date of acceptance). If your application is refused it remains possible to reapply in the future.
-
-
 ## Next steps after DACO Approval
 
 Once your DACO application has been approved, log in to the **[ICGC Data Portal](https://dcc.icgc.org/)** using the **same Google email address (OpenID)** that is associated with your DACO account.
```

Comment:
@hknahal let's get rid of this because there is no section like that anymore
Please note, if you intend to access controlled data in the Cloud, please ensure you complete the "Cloud Storage Access Agreement" section of your DACO application. Otherwise, you will see a red cloud icon.